### PR TITLE
[wasm] Set WasmEnableWebcil to true by default

### PR DESCRIPTION
**Pull request by lambdageek**
Switch to webcil by default in WebAssembly.

Contributes to https://github.com/dotnet/runtime/issues/80807

Also fix some debugger test failures.

### CODE REVIEW

File: docs/design/mono/webcil.md [hunk 1]
```diff
@@ -2,21 +2,83 @@
...
+    local.get $d
+    i32.const 0
+    local.get $n
+    memory.init 1))
+```
+
+That is, the module imports linear memory 0 and exports:
+* a global `i32` `webcilVersion` encoding the version of the WebAssembly wrapper (currently 0),
+* a function `getWebcilSize : i32 -> ()` that writes the size of the Webcil payload to the specified
+  address in linear memory as a `u32` (that is: 4 LE bytes).
```
Comment (teo-tsirpanis) [inline docs/design/mono/webcil.md:45]: Why not just return the size?

Comment (teo-tsirpanis) [reply docs/design/mono/webcil.md:45]: Thanks, I just saw the rationale sections below. 😅

**Comment by lambdageek** [comment]
the symbol server debugger tests are failing with a pdb checksum mismatch.  which is weird because we're apparently reading the debug directory data correctly.  but somehow it is wrong

**Comment by lambdageek** [comment]
@maraf @radical this is ready for review. the multi-threaded debugger issues are known issues. single-threaded debugging is passing with webcil-in-wasm now

**Review by maraf — APPROVED** [review]
Looks good to me 👍

**Comment by lambdageek** [comment]
@maraf will work on downstream test fixes for dotnet/sdk and dotnet/aspnetcore.  We're going to merge this PR.

**Pull request merged**